### PR TITLE
Basic devices API implementation

### PR DESCRIPTION
**Pull request by packituz-gttz**
Endpoints:
- Device

Models:
- Devices
- Products

Each endpoint has tests. :smiley_cat:

### CODE REVIEW

File: devices/config.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Configure the SqlAlchemy part of the app instance
+# app.config["SQLALCHEMY_ECHO"] = True
+app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get('SQLALCHEMY_DATABASE_URI', postgres_url)
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+# Create the SqlAlchemy db instance
+db = SQLAlchemy(app)
+
+from models import Product, Device
```
Comment (erikriver) [inline devices/config.py]: Porqué se importa esto aquí?

File: Pipfile [hunk 2]
```diff
@@ -8,7 +8,8 @@ verify_ssl = true
 [packages]
```
Comment (erikriver) [inline Pipfile:8]: Move Pipfile inside `devices` folder, each microservice have their own environment isolated.

File: Pipfile [hunk 3]
```diff
@@ -8,7 +8,8 @@ verify_ssl = true
...
 flask = "*"
 flask-sqlalchemy = "*"
-connexion = "*"
+connexion = {extras = ["swagger-ui"],version = "*"}
+psycopg2 = "*"
 
 [requires]
 python_version = "3.6"
```
Comment (erikriver) [inline Pipfile:17]: I suggest use the last Python version, like 3.7

File: devices/views.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+from flask import abort
```
Comment (erikriver) [inline devices/views.py:1]: In a MVC pattern, I think those are controllers instead views.

File: Pipfile (hunk 2 above)
Comment (packituz-gttz) [reply Pipfile:8]: OK, I have made the changes

**Review by erikriver — APPROVED** [review]

**Pull request merged**